src/BaselineOfFamix/BaselineOfFamix.class.st

```diff
@@ -108,7 +109,7 @@ BaselineOfFamix >> baseline: spec [
 				group: 'BasicTraits' with: #('Famix-Traits');
 				group: 'EntitiesCompatibility' with: #('Famix-Compatibility-Entities');
 				group: 'ModelCompatibility' with: #('Famix-Compatibility-Generator' 'EntitiesCompatibility');
-				group: 'EntitiesJava' with: #('Famix-Java-Entities');
+				group: 'EntitiesJava' with: #('Famix-Java-Entities' 'Famix-Java-Visitor');
```

Comment:
Is it the correct way to add the Visitor. Maybe it should be in another group (so one can load the model without the visitor)

Reply:
Something like:
```smalltalk
group: 'EntitiesJava' with: #('Famix-Java-Entities');
group: 'EntitiesJavaWithVisitor' with: #('EntitiesJava' 'Famix-Java-Visitor');
```
We could also add 
```smalltalk
group: 'ModelJavaWithVisitor' with: #('Famix-Java-Generator' 'EntitiesJavaWithVisitor');
```

What do you think ?

Reply:
I like the first one
No need of the last one IMO